### ISSUE TITLE
oauth2: token expired and refresh token is not set

### ISSUE THREAD

**Issue by dougfultz** [google/cups-connector#208]
My printer is showing offline in Google Cloud print and when I attempt to show the printer status with cups-connector I get the following error:

```
gcp@HOST:~$ /opt/gcp_cups_connector/gcp-cups-connector-util show-gcp-printer-status
POST failure: Post https://www.google.com/cloudprint/printer: oauth2: token expired and refresh token is not set
```

I am unable to find any command to refresh the token.

**Comment by dougfultz** [comment]
I initialized the config file again, which caused a duplicate but working entry in Google Cloud Print.  Is it possible to simply refresh the oath token without creating new printer entries?

**Comment by dougfultz** [comment]
Yes, both the "old" config and the new config has a `robot_refresh_token` entry.

**Comment by jay0lee** [comment]
In your original command, you didn't specify the --config-filename so the refresh token couldn't be found. Delete the new  config and printer it generated and use --config-filename to specify the config file which contains the refresh token.

**Comment by dougfultz** [comment]
```
gcp@HOST:~$ /opt/gcp_cups_connector/gcp-cups-connector-util --config-filename /etc/gcp_cups_connector/gcp-cups-connector.config.json.bak show-gcp-printer-status
https://www.google.com/cloudprint/printer call failed: Printer Id required for this request.
```

**Comment by jay0lee** [comment]
--gcp-printer-id is also required for this command. Get the printer ID from Google.com/cloudprint website.

**Comment by dougfultz** [comment]
Is this documented somewhere?  The output doesn't exactly make it clear that another flag is required and I don't recall seeing that flag as an option in the "help" output.

**Comment by kylemacfarlane** [comment]
I had a similar problem getting `show-gcp-printer-status` to work. My problem is that when I provide `--config-filename` it errors out with "incorrect usage" and says you must only provide `--printer-id`. Thus the only way to get printer status is by first cding into `/etc/gcp_cups_connector` so that the default config argument is valid.
